Documentation/contributor-guide/roadmap.md

```diff
@@ -0,0 +1,47 @@
+# roadmap
+
+etcd uses GitHub milestones to track all tasks in each major or minor release. The roadmap.md file only records the most
+important tasks for each release.
+Please also refer to [here](https://docs.google.com/spreadsheets/d/1hFpIbkI0IDum0SPy8txj7bVHxQRrAB_82f1Yqz_Pa8M/edit#gid=0&fvid=1685003197)
+for more context. Note that etcd will continue to focus on technical debt over the next few major or minor releases.
```

Comment:
> Note that etcd will continue to focus on technical debt over the next few major or minor releases.

This could be to hard for other contributors I would try to assign a theme for each release. For example for v3.6 we can go with "Correctness and technical depth" (robustness tests, fixing live downgrades and v2 storage removal as major deliverables), but for v3.7 we could be more ambitious with "performance" to motivate people getting through v3.6 faster.

Reply:
I do not see enough confidence to be more ambitious given the technical debt included in this roadmap. But of course, as discussed above, the list may shift over time.

Reply:
Sure, just wanted to point out that you have 2 performance tasks in v3.7